install/Makefile

```diff
@@ -142,6 +143,13 @@ export HUGO_VERSION ?= 0.45.1
 hugo: hugo-$(OS)
 	@exit 0
 
+export SHELLCHECK_VERSION ?= 0.5.0
+# Releases: https://github.com/koalaman/shellcheck/releasess
+## ShellCheck, a static analysis tool for shell scripts 
```

Comment:
Lets keep the same formatting as all the other targets.

![image](https://user-images.githubusercontent.com/52489/44046492-40ae6554-9ef1-11e8-8f6f-7f842b3cffd3.png)

Reply:
> Install shellcheck static analysis tool for shell scripts